### DISCUSSION TITLE
There was no response from GitHub Support?

### DISCUSSION THREAD

**Discussion by mistletoeKANO** [community/community#28832] [New to GitHub]
My account was flagged 4 days ago. i have try to send help to github support, but no any response. anyone can help me? i can't work now, please!  my ticket [Ticket 1732030](https://support.github.com/ticket/1732030)

**Comment by Frid0l1n** [comment]
Have you tried to open a new ticket and rewrite your problem. I had the same issue and i made a new ticket it worked after the second ticket.

**Comment by mistletoeKANO** [comment]
Thank u
